ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx

```diff
@@ -1,10 +1,22 @@
 import { STATUS_LIST } from '@/constants';
 import { intl } from '@/utils/intl';
 import { findByValue } from '@oceanbase/util';
-import { Card, Col, Table, Tag } from 'antd';
+import { Button, Card, Col, Modal, Table, Tag } from 'antd';
 import type { ColumnsType } from 'antd/es/table';
+import { flatten } from 'lodash';
+
+export default function ServerTable({
+  clusterDetail,
+}: {
+  clusterDetail: API.ClusterDetail[];
+}) {
+  const servers = clusterDetail?.servers;
+  // 是否为单节点集群： zone的个数 ===1 && OBServer的个数 == 1
```

Comment:
The server that is the only one server in a zone can not be deleted.

Reply:
done